addons/payment_stripe/models/payment_transaction.py

```diff
@@ -138,7 +138,7 @@ def _stripe_create_customer(self):
                 'description': f'Odoo Partner: {self.partner_id.name} (id: {self.partner_id.id})',
                 'email': self.partner_email or None,
                 'name': self.partner_name,
-                'phone': self.partner_phone or None,
+                'phone': self.partner_phone[0:20] or None,
```

Comment:
```suggestion
                'phone': self.partner_phone and self.partner_phone[0:20] or None,
```
If no partner_phone, you cannot slice `False`